tests/src/JIT/Regression/JitBlue/GitHub_23199/GitHub_23199.csproj

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I will check.

Reply:
It takes less than a second. It needs 0xc for the first test case but will fail with 4 on the other.

Moved to pri1.

Reply:
It seems very scary to have a test that forces gc stress.

Reply:
I need to keep `COMPlus_GcStressOnDirectCalls` for this test and it will be strange to have it without `COMPlus_GcStress`, won't it?

Reply:
IMO, it's not a problem to force GCStress as long as the test runs in less than a second or two (on all platforms where it is enabled).

Unfortunately, GcStressOnDirectCalls is not available in retail (non-DEBUG) builds, so your test doesn't really force the behavior you are testing in that scenario.

Reply:
Do we know why `GcStressOnDirectCalls` is not on by default?

Reply:
There is a problem when we use native images and can hit AV because direct call target could be invalid before eager fixups are applied.

There is a comment about that:
https://github.com/dotnet/coreclr/blob/7e17986ff71a70ba328895e0b99e1de83d1bf5b5/src/vm/gccover.cpp#L758-L775

Reply:
That may be a fragile NGEN restriction, though.

Reply:
Can be, there is an issue about that already, https://github.com/dotnet/coreclr/issues/4220
cc @swaroop-sridhar